src/main/java/seedu/address/model/session/SessionDate.java

```diff
@@ -60,6 +61,20 @@ public LocalTime getTime() {
         return dateTime.toLocalTime();
     }
 
+    public static boolean hasSameTime(SessionDate s1, SessionDate s2) {
+        return s1.getTime().equals(s2.getTime());
```

Comment:
Actually not sure if other groups will nitpick this, but we could give s1 and s2 a more meaningful name such as sessionDate1 and sessionDate2. Will be longer, but at least we prevent other group from nitpicking these small bugs ah.

Reply:
Yep thanks. actualy given your comments I think there was also no need for this to be a static method.
Will replace with a .equalTime(SessionDate sessionDate)